src/features/click-to-play.js

```diff
@@ -1,1034 +1,27 @@
 import { createCustomEvent, sendMessage, OriginalCustomEvent, originalWindowDispatchEvent } from '../utils.js'
 import {
-    logoImg, loadingImages, closeIcon, blockedFBLogo, ddgFont, ddgFontBold,
-    blockedYTVideo, videoPlayDark, videoPlayLight
-} from '../assets/ctl-assets.js'
+    logoImg, loadingImages, closeIcon
+} from './click-to-play/ctl-assets.js'
+import { styles, getConfig } from './click-to-play/ctl-config.js'
 
 let devMode = false
 let isYoutubePreviewsEnabled = false
 let appID
 
 const titleID = 'DuckDuckGoPrivacyEssentialsCTLElementTitle'
 
+/*********************************************************
+ *  config set by getConfig() at init per localization
+ *********************************************************/
+let config = {}
+let sharedStrings = {}
+
 // TODO: Remove these redundant data structures and refactor the related code.
 //       There should be no need to have the entity configuration stored in two
 //       places.
 const entities = []
 const entityData = {}
 
-/*********************************************************
```

Comment:
Would you mind splitting the changes up, so that the first PR is to just move the config into the separate file? I think that would make the rest of the changes a lot easier to review.

Reply:
That makes sense, though lets resolve the other conversation first so this will fit whatever pattern we decide on.